static/css/dd.css

```diff
@@ -1,95 +1,97 @@
 #lists {background-color: #aaaaaa;}
 
 #boxdiv {width:400px;height:30px;padding:10px;border:3px solid #aaaaaa; margin-bottom:30px; background-color:#FFCC99}
-     
```

Comment:
We should publish and adhere to some kind of style guide, so we don't have these semantically meaningless changes introduced into the history.

Reply:
Good point. I missed re-styling this one, unfortunately.